src/protocols/MeterSML.cpp

```diff
@@ -321,9 +328,18 @@ bool MeterSML::_parse(sml_list *entry, Reading *rd) {
 			rd->value(sml_value_to_double(entry->value) * pow(10, scaler));
 		}
 
+		if (_direction_from_status && entry->status) {
+			*direction_from_status = (*(entry->status->data.status16) & (1 << 5)) == (1 << 5);
+		}
+
 		ReadingIdentifier *rid(new ObisIdentifier(obis));
 		rd->identifier(rid);
 
+		if (_direction_from_status && *direction_from_status &&
+			obis == Obis(1, 0, 15, 7, 0, 0xff)) {
```

Comment:
is it OK to hardcode this?
will we ever want to apply the direction to any other reading?